.github/ISSUE_TEMPLATE/10-issue_report.yaml

```diff
@@ -0,0 +1,136 @@
+name: Report a Bug
+title: "<title>"
+description: |
+  You found a bug in Bareos causing the application to crash or
+  throw an exception, a component is buggy, or something looks wrong.
+labels: needs triage
+body:
+body:
```

Comment:
Why 2 bodies ?

Reply:
Thanks good catch, the linter didn't catch it :-)